src/modules/google-tag-manager/hooks/afterRegistration.ts

```diff
@@ -0,0 +1,100 @@
+import { currentStoreView } from '@vue-storefront/core/lib/multistore'
+
+export function afterRegistration(Vue, config, store, isServer) {
+  if (config.analytics.id && !isServer) {
```

Comment:
this should check for `config.googleTagManager.id`

Reply:
Good catch, Thanks